parcel/ParcelModel.jl

```diff
@@ -112,16 +123,17 @@ function parcel_model(dY, Y, p, t)
 
     # number concentration and ...
     dNᵢ_dt = dNᵢ_dt_dep + dNᵢ_dt_imm + dNᵢ_dt_hom
-    dNₐ_dt = -dNᵢ_dt_dep
-    dNₗ_dt = -dNᵢ_dt_imm - dNᵢ_dt_hom
+    dNₐ_dt = -dNᵢ_dt_dep - dNₗ_dt_act
+    dNₗ_dt = dNₗ_dt_act - dNᵢ_dt_imm - dNᵢ_dt_hom
     # ... water mass budget
+    dqₗ_dt_a2l = dqₗ_dt_act
     dqₗ_dt_v2l = dqₗ_dt_ce
     dqᵢ_dt_l2i = dqᵢ_dt_imm + dqᵢ_dt_hom
     dqᵢ_dt_v2i = dqᵢ_dt_dep + dqᵢ_dt_ds
 
     # Update the tendecies
     dqᵢ_dt = dqᵢ_dt_v2i + dqᵢ_dt_l2i
-    dqₗ_dt = dqₗ_dt_v2l - dqᵢ_dt_l2i
+    dqₗ_dt = dqₗ_dt_v2l - dqᵢ_dt_l2i + dqₗ_dt_a2l
```

Comment:
Should there be a corresponding sink to water vapor as well?

Reply:
Because we are neglecting the condensation onto the aerosol, the `dqₗ_dt_a2l` term only accounts for the aerosol and not any of the water vapor. So technically what it's doing there is converting a prescribed volume of soluble aerosol into a volume of liquid water. A corresponding sink would be a decrease in "q_a" but since mass of aerosols is not tracked, it's not in the parcel anywhere. I could add it though if that's something we want?

Reply:
In that case, shouldn't it be multiplied by the aerosol density and not liquid water?

`dqₗ_dt_act = dNₗ_dt_act * 4 * FT(π) / 3 * r_nuc^3 * ρₗ / ρ_air`

Maybe it would be better to not mix the aerosol mass and water mass. We could assume some sort of initial cloud droplet size after activation, and compute the corresponding sink of water vapor based on that